testdata/source/generic_importer/test_dummy_predictor/import_results.beancount

```diff
@@ -0,0 +1,65 @@
+;; date: 2020-01-01
+;; info: {"filename": "<testdata>/csv/generic_statement.csv", "line": 1, "type": "text/csv"}
+
+; features: []
+2020-01-01 * "by debit card-OTHPG 063441 GOOGLE CLOUD INDIA PVTTHANE-"
+  Assets:Bank   -1 USD
+    date: 2020-01-01
+    source_desc: "by debit card-OTHPG 063441 GOOGLE CLOUD INDIA PVTTHANE-"
+  Assets:Dummy   1 USD
+    unconfirmed_account: ""
```

Comment:
It looks like the test output needs to be updated for the change to `True`.

Reply:
Done